vue-frontend/src/components/home-new/ContributionSection.vue

```diff
@@ -1,41 +1,41 @@
 <template>
   <div
-    class="tw-container lg:tw-flex lg:tw-flex-row-reverse tw-justify-center tw-max-w-[1878px] lg:tw-pt-20 tw-text-center lg:tw-text-left"
+    class="tw-relative xll:tw-container tw-my-0 tw-mx-auto tw-w-[90%] xll:tw-max-w-[1500px] tw-p-0"
```

Comment:
xll is working?

Reply:
Yes, we had added this breakpoint in tailwind config